sample-dcrwallet.conf

```diff
@@ -206,3 +206,6 @@
 
 ; Proportion of funds to leave in wallet when stake mining
 ; ticketbuyer.balancetomaintainrelative=0.3
+
+; Purchase tickets using a split transaction if this value is 1 or number of eligible credits is greater than or equal to this
```

Comment:
Don't have an alternative yet for this but I think it could be reworded/simplified.